### ISSUE TITLE
Dev 1Password reorganization

### ISSUE THREAD

**Issue by chiaberry** [cityofaustin/atd-data-tech#11283]
Audit the Developers vault in 1password

Add tags to the entries
* vision zero
* moped
* micromobility (?) 
* [any other tags that make sense]

Come up with a list of entries that should be archived, we can review the list as a team.

**Comment by dianamartin** [comment]
@chiaberry I see this might be affected when we move 1password environments.. see my slack post from 3/20 on the "real-talk" channel.

**Comment by dianamartin** [comment]
@frankhereford @chiaberry this is done, right?

**Comment by chiaberry** [comment]
@dianamartin no, we have not worked on auditing or comprehensively tagging our entries in the dev vault
